.reuse/dep5

```diff
@@ -11,6 +11,10 @@ Files: .clang-format
 Copyright: 2020 RizinOrg <info@rizin.re>
 License: LGPL-3.0-only
 
+Files: src/cp/.clang-format
```

Comment:
Maybe move this out of cp and make it global in the root dir?

Reply:
Ok 👍

Reply:
yes you can submit this in another PR where you patch all the diffs.